sopel/modules/safety.py

```diff
@@ -193,17 +196,21 @@ def url_handler(bot, trigger):
     if positives > 1:
         # Possibly malicious URL detected!
         confidence = '{}%'.format(round((positives / total) * 100))
-        msg = 'link posted by %s is possibly malicious ' % bold(trigger.nick)
+        msg = (
+            'link posted by %s is possibly malicious '
+            % formatting.bold(trigger.nick)
+        )
         msg += '(confidence %s - %s/%s)' % (confidence, positives, total)
-        bot.say('[' + bold(color('WARNING', 'red')) + '] ' + msg)
+        bot.say(WARNING + ' ' + msg)
```

Comment:
This isn't exactly equivalent to previous behavior. If you're dropping the square brackets around "WARNING", it should at least have a colon:

```suggestion
        bot.say(WARNING + ': ' + msg)
```

Though maybe the colon should be in bold/red, which means it'd need to be part of the constant above. Which begs the question, why define a constant for only one use of the formatted string?

Reply:
It's generated once because unlike a simple str, it is generated with multiple function calls. I guess it doesn't really matter here, it's not called often. I guess I wasn't sure what to do with that, and the end result is far from what I intended at first.

Reply:
Fixed it.